app/dashboard/page.tsx

```diff
@@ -20,6 +21,7 @@ export default async function DashboardPage() {
       <div>
         <div className="text-text px-4 py-6 flex justify-between">
           <DashboardGreeting username={username} />
+          <DashboardStats />
```

Comment:
This section looks bad:

<img width="681" alt="image" src="https://github.com/user-attachments/assets/3f2d261b-212b-47d7-833a-5bfb73c5b159" />

We want them stacked, which should be an easy fix.